mentors/javascript/a.json

```diff
@@ -34,15 +34,15 @@
   {
     "github_username": "alexZielonko",
     "name": "Alex Zielonko",
-    "link_text": "github.com/alexZielonko",
+    "link_text": "GitHub.com/alexZielonko",
     "link_url": "https://github.com/alexZielonko",
     "avatar_url": "https://avatars1.githubusercontent.com/u/5274697?s=460&v=4",
     "bio": "I'm one part professional developer and two parts programming enthusiast, shaken with a liberal dash of pragmatism."
   },
   {
     "github_username": "aibrahim3546",
     "name": "Ahmad Ibrahim",
-    "link_text": "https://aibrahim3546.github.io/",
+    "link_text": null,
```

Comment:
These are Not equivalent, per readme null means take this fields value from GitHub.

Reply:
I was working under the assumption that if "link_text" is null that "link_url" would be displayed instead.

Reply:
I think I am going to modify my changes to `Rakefile` in #791 to effectively ignore what is in "link_text" if it is identical to what is in "link_url".  This would make some of the changes I have made in this PR unnecessary.